scripts/batch_list.js

```diff
@@ -0,0 +1,58 @@
+months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"]

```

Comment:
There's got to be a library for this instead of hard coding it...what if the months change??